### PR TITLE
i18n(es): `sitemap.mdx` update

### PR DESCRIPTION
**Pull request by Mardroide**
#### What kind of changes does this PR include?
- Updated content

#### Reference
- [3af1498](https://github.com/withastro/docs/commit/3af14984f4e7a9fa476bcbd501d7a1516bf106c8)

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/sitemap.mdx [hunk 1]
```diff
@@ -18,7 +18,9 @@ Un Sitemap es un archivo XML que describe todas las páginas, videos y archivos
 
 Se recomienda utilizar un archivo Sitemap para sitios grandes con múltiples páginas. Si no utilizas un Sitemap, la mayoría de los motores de búsqueda aún podrán listar las páginas de tu sitio, pero un Sitemap es una excelente manera de asegurarte de que tu sitio sea lo más amigable posible para los motores de búsqueda.
 
-Con Astro Sitemap, no tienes que preocuparte por crear este archivo: construye tu sitio Astro como lo harías normalmente y la integración de Astro Sitemap rastreará tus rutas y creará el archivo del Sitemap.
+Con Astro Sitemap, no tienes que preocuparte por crear este archivo XML por ti mismo: la integración Astro Sitemap rastreará tus rutas generadas estáticamente y creará el archivo sitemap, incluyendo [rutas dinámicas](/es/core-concepts/routing/#dynamic-routes) como `[...slug]` o `src/pages/[lang]/[version]/info.astro` generado por `getStaticPaths()`.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/sitemap.mdx:21]:
```suggestion
Con Astro Sitemap, no tienes que preocuparte por crear este archivo XML tú mismo: la integración Astro Sitemap rastreará tus rutas generadas estáticamente y creará el archivo sitemap, incluyendo [rutas dinámicas](/es/core-concepts/routing/#dynamic-routes) como `[...slug]` o `src/pages/[lang]/[version]/info.astro` generado por `getStaticPaths()`.
```

File: src/content/docs/es/guides/integrations-guide/sitemap.mdx [hunk 2]
```diff
@@ -18,7 +18,9 @@ Un Sitemap es un archivo XML que describe todas las páginas, videos y archivos
...
+
+Esta integración no puede generar entradas de mapa del sitio para rutas dinámicas en [modo SSR](/es/guides/server-side-rendering/).
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/sitemap.mdx:23]:
```suggestion
Esta integración no puede generar entradas de sitemap para rutas dinámicas en [modo SSR](/es/guides/server-side-rendering/).
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR, te dejo unas sugerencias

File: src/content/docs/es/guides/integrations-guide/sitemap.mdx [hunk 3]
```diff
@@ -18,7 +18,9 @@ Un Sitemap es un archivo XML que describe todas las páginas, videos y archivos
 
 Se recomienda utilizar un archivo Sitemap para sitios grandes con múltiples páginas. Si no utilizas un Sitemap, la mayoría de los motores de búsqueda aún podrán listar las páginas de tu sitio, pero un Sitemap es una excelente manera de asegurarte de que tu sitio sea lo más amigable posible para los motores de búsqueda.
 
-Con Astro Sitemap, no tienes que preocuparte por crear este archivo: construye tu sitio Astro como lo harías normalmente y la integración de Astro Sitemap rastreará tus rutas y creará el archivo del Sitemap.
+Con Astro Sitemap, no tienes que preocuparte por crear este archivo XML tú mismo: la integración Astro Sitemap rastreará tus rutas generadas estáticamente y creará el archivo sitemap, incluyendo [rutas dinámicas](/es/core-concepts/routing/#dynamic-routes) como `[...slug]` o `src/pages/[lang]/[version]/info.astro` generado por `getStaticPaths()`.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/sitemap.mdx:21]: tienes que arreglar el link roto

Comment (Yan-Thomas) [inline src/content/docs/es/guides/integrations-guide/sitemap.mdx:21]:
```suggestion
Con Astro Sitemap, no tienes que preocuparte por crear este archivo XML tú mismo: la integración Astro Sitemap rastreará tus rutas generadas estáticamente y creará el archivo sitemap, incluyendo [rutas dinámicas](/es/core-concepts/routing/#rutas-dinámicas) como `[...slug]` o `src/pages/[lang]/[version]/info.astro` generado por `getStaticPaths()`.
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM, thanks peeps! Went ahead and fixed the link for ya 🙌

**Pull request merged**